R/gains-and-losses.R

```diff
@@ -1,3 +1,26 @@
+#' Helper function to clean up code which generates realized gains and losses.
+#'
+#' @param json_response the API response in JSON format.
+#' @param short_or_long one of either "short" or "long" for the term in which gains are evaluated.
+#'
+get_gain_loss_by_term <- function(
+    json_response,
+    short_or_long = c("short", "long")
+) {
+  short_or_long <- switch(
+    short_or_long,
+    "short" = "stGainLoss",
+    "long" = "ltGainLoss"
+  )
+
+  json_response |>
+    pluck("lot") |>
+    map(
+      ~pluck(.x, short_or_long)
+    ) |>
+    unlist()
```

Comment:
Did a few quick changes like this where there's no real need to define an object and then return it.